### PR TITLE
[1155] Liner

### PR DESCRIPTION
**Pull request by Sybrand**
Liners & stacking code

DO NOT MERGE - pending merge of 1245 by Gary.

### CODE REVIEW

File: app/backend/wells/models.py [hunk 1]
```diff
@@ -1285,14 +1286,15 @@ class Screen(AuditModel):
...
-                                    blank=False, null=True, validators=[MinValueValidator(Decimal('0.01'))])
+    activity_submission = models.ForeignKey(ActivitySubmission, db_column='filing_number',
+                                            on_delete=models.CASCADE, blank=True, null=True,
+                                            related_name='screen_set')
+    well = models.ForeignKey(Well, db_column='well_tag_number', on_delete=models.CASCADE, blank=True,
+                             null=True, related_name='screen_set')
+    start = models.DecimalField(db_column='screen_from', max_digits=7, decimal_places=2, verbose_name='From',
+                                blank=True, null=True, validators=[MinValueValidator(Decimal('0.00'))])
+    end = models.DecimalField(db_column='screen_to', max_digits=7, decimal_places=2, verbose_name='To',
+                              blank=False, null=True, validators=[MinValueValidator(Decimal('0.01'))])
```
Comment (stephenhillier) [inline app/backend/wells/models.py]: Not in your scope for this PR but this `validators=` code is a bit odd.  There must be a better way to handle "the end value should be more than the start value`

File: app/backend/wells/urls.py [hunk 2]
```diff
@@ -24,6 +24,9 @@
         views.WellDetailView.as_view(), name='well_detail'),
 
     # API endpoints
+    # Well
+    url(r'^api/v1/well/(?P<well_tag_number>[0-9]+)$',
```
Comment (stephenhillier) [inline app/backend/wells/urls.py]: plural: `api/v1/wells/ `

File: app/frontend/src/submissions/components/SubmissionForm/Liner.vue [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+            :loaded="fieldsLoaded['liner_to']"
+            />
+        </b-col>
+      </b-row>
+      <b-row>
+        <b-col>Liner Perforations</b-col>
+      </b-row>
+      <b-row>
+        <b-col>
+          <table class="table">
```
Comment (stephenhillier) [inline app/frontend/src/submissions/components/SubmissionForm/Liner.vue]: `<table class="table table-sm">` (per Nicole)

File: app/backend/wells/models.py (hunk 1 above)
Comment (Sybrand) [reply app/backend/wells/models.py]: Yeah, I guess we'll look at it when we do validation.

**Comment by Sybrand** [comment]
[EPIC#Submission | Liner Information |5| !!! [BV: 35]](https://trello.com/c/Ap5lbxVo/1155-epicsubmission-liner-information-5-bv-35)

**Pull request merged**